src/main/java/org/humancellatlas/ingest/protocol/Protocol.java

```diff
@@ -20,9 +19,4 @@ public Protocol(Object content) {
         super(EntityType.PROTOCOL, content);
     }
 
-    public Protocol addToSubmissionEnvelope(SubmissionEnvelope submissionEnvelope) {
```

Comment:
Is this now handled by the super (MetaDataDocument)?

Reply:
we don't need this method as we aren't adding to a list of subEnvs anymore. Instead we setSubmissionEnvelope. Note that this property now refers to which subEnv the doc is contained. The project-submission relation needs to be captured. See prev comments.